tools/test_runner.js

```diff
@@ -18,6 +18,8 @@ var console_wrapper = require('common_js/module/console');
 var builtin_modules =
   Object.keys(process.native_sources).concat(Object.keys(process.binding));
 
+var default_timeout = 240; // (in seconds)
```

Comment:
That is a pretty long timeout. A bad patch (where all tests are timed out) could make the test system run kind of forever.